### DISCUSSION TITLE
part-ii/2013-39/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#281] [Question comments]
Part II 2013 Differential Geometry: Paper 2, Section II, H 

https://questions.tripos.org/part-ii/2013-39/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(d) Consider the circle at the top/bottom of the torus. The surface normal is perpendicular to the curve normal, so the normal curvature vanishes. The Gaussian curvature is zero at the top and the bottom. 

</details>
